packages/dapp/src/components/Dashboard/ProposalChallengeSummary.tsx

```diff
@@ -0,0 +1,137 @@
+import * as React from "react";
+import { Link } from "react-router-dom";
+
+import { canParamPropChallengeBeResolved, didParamPropChallengeSucceed } from "@joincivil/core";
+import {
+  UserVotingSummary,
+  CHALLENGE_RESULTS_VOTE_TYPES,
+  StyledDashbaordActvityItemSection,
+  StyledDashbaordActvityItemSectionOuter,
+  StyledDashbaordActvityItemHeader,
+  StyledDashbaordActvityItemSectionInner,
+  StyledChallengeSummarySection,
+  StyledDashboardActivityItemAction,
+  DashboardActivityProposalItemCTAButton,
+} from "@joincivil/components";
+import { getFormattedTokenBalance } from "@joincivil/utils";
+
+import { WinningChallengeResults } from "./WinningChallengeResults";
+import { MyTasksProposalItemSubComponentProps } from "./MyTasksProposalItem";
+
+const CurrentChallengeStateExplanation: React.SFC<MyTasksProposalItemSubComponentProps> = props => {
+  const { challenge } = props;
+
+  if (!challenge) {
+    return <></>;
+  }
+
+  const isResolved = challenge.resolved;
+  const canResolveChallenge = canParamPropChallengeBeResolved(challenge);
+  const didChallengeSucceed = didParamPropChallengeSucceed(challenge);
+
+  const currentProposalStatus = didChallengeSucceed ? "reject" : "approve";
+
+  // The ordering of these cases is significant. Appeal Challenge, then Appeal, the Original Challenge states should be checked in that order to ensure the proper explanation is set.
+  let explanation = <></>;
+  if (canResolveChallenge || isResolved) {
+    explanation = <p>The Civil Community voted to {currentProposalStatus} this parameter proposal.</p>;
+  }
+
+  return explanation;
+};
+
+const ChallengeSummary: React.SFC<MyTasksProposalItemSubComponentProps> = props => {
+  const { challengeID, challenge, proposalUserChallengeData, showClaimRewardsTab, showRescueTokensTab } = props;
+
+  if (!proposalUserChallengeData || !challenge) {
+    return <></>;
+  }
+
+  // const isResolved = challenge.resolved;
+  // const canResolveChallenge = canParamPropChallengeBeResolved(challenge);
+  // const didChallengeSucceed = didParamPropChallengeSucceed(challenge);
+
+  const {
+    // canUserReveal,
+    canUserCollect,
+    canUserRescue,
+    // didUserCommit,
```

Comment:
what's up with these commented out lines?

Reply:
These were some leftovers from the non-Proposal Challenge version of this file. I'll clean these out and restore them in a future iteration, if needed.